osquery/tables/system/linux/portage.cpp

```diff
@@ -0,0 +1,416 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ *  Portage support by J.O. Aho <trizt@aho.hk>
+ */
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/algorithm/string/join.hpp>
+#include <boost/algorithm/string/predicate.hpp>
+#include <boost/algorithm/string/trim_all.hpp>
+#include <boost/concept_check.hpp>
+#include <boost/filesystem/operations.hpp>
+#include <boost/filesystem/path.hpp>
+
+#include <osquery/core.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+
+#define fs boost::filesystem
+
+namespace osquery {
+namespace tables {
+
+/* File paths used by portage */
+fs::path portagePackageDir = "/var/db/pkg";
```

Comment:
Let's use `const std::string`s for these configuration options.

For global-constant objects we prefix the lower-camel-case name with a `k`. So this becomes `kPortagePackageDir`. Also, consider using uniform initialization syntax:
```cpp
const std::string kPortagePackageDir{"/var/db/pkg"};
```